app/helpers/application_helper.rb

```diff
@@ -1,2 +1,37 @@
 module ApplicationHelper
+  # This method makes it easier to yield content with a default value.
+  #
+  # USAGE:
+  # ```
+  # <% yield_with_default_for :signature do %>
+  #   Default signature here
+  # <% end %>
+  # ```
+  #
+  # The same functionality can be achieved with the following code:
+  # ```
+  # <% if content_for?(:signature) %>
+  #   <%= yield :signature %>
+  # <% else %>
+  #   Default signature here
+  # <% end %>
+  # ```
+  #
+  # In helpers, `content_for` is used instead of `yield`.
+  #
+  # For more information regarding `yield` and `content_for`, see:
+  # - https://guides.rubyonrails.org/layouts_and_rendering.html#understanding-yield
+  # - https://apidock.com/rails/ActionView/Helpers/CaptureHelper/content_for
+  # - https://apidock.com/rails/v5.2.3/ActionView/Helpers/CaptureHelper/content_for%3F
+  def yield_with_default_for(key, &block)
```

Comment:
```suggestion
  def yield_or_content_for(key, &block)
```

Actually, this is probably clearer, but feel free to disregard 😆